stack/base/Dockerfile

```diff
@@ -52,6 +52,13 @@ if [ -f /opt/bin/load-singlesshagent.sh ]; then\n\
    . /opt/bin/load-singlesshagent.sh\n\
 fi\n' >> "/home/${NB_USER}/.bashrc"
 
+# Add ~/.local/bin to PATH where the dependencies get installed via pip
+RUN echo "export PATH=\${PATH}:/home/${NB_USER}/.local/bin" >> "/home/${NB_USER}/.bashrc"
+
+# Workaround for missing Jupyter config path,
+# needed for locally installed notebook extensions (e.g. nglview)
+ENV JUPYTER_CONFIG_PATH=/home/${NB_USER}/.local/etc/jupyter
```

Comment:
Hi @danielhollas, this will override original `JUPYTER_CONFIG_PATH` if it was set? This is a search path so can also be like `$PATH` and set as `${JUPYTER_CONFIG_PATH}:/home/${NB_USER}/.local/etc/jupyter`? Can you test it?

Reply:
I don't think this path is set, but I agree it's better to do it as you propose, thanks.